src/fr/styles/typographie/typographie.md

```diff
@@ -81,11 +81,11 @@ Remarque : La propriété de la police représente à la fois l'épaisseur, la t
 
 ### Familles de police
 
-Les en-têtes font appel à la police «&nbsp;Lato&nbsp;». Les paragraphes et autres éléments textuels se servent de la police «&nbsp;Noto Sans&nbsp;».
+Les titres font appel à la police «&nbsp;Lato&nbsp;». Les paragraphes et autres éléments textuels se servent de la police «&nbsp;Noto Sans&nbsp;».
 
 La famille de police comprend des valeurs de rechange. Les valeurs de rechange sont seulement utilisées pour une famille de police lorsqu'une police n'est pas disponible.
 
-- N'utilisez seulement les unités de style en-tête que pour les en-têtes. Tout autre contenu peut utiliser la police Texte.
+- N'utilisez seulement les unités de style titre que pour les titres. Tout autre contenu peut utiliser la police Texte.
```

Comment:
I noticed a syntax error that was there previously. In addition to your change, "seulement" should be deleted. It should read:

`N'utilisez les unités de style titre que pour les titres.`